tests/Integration/data/helpers.php

```diff
@@ -17,3 +17,8 @@ function retryAcceptsCallableAsSleepMilliseconds(): void
         return true;
     });
 }
+
+function envReturnsGenericValue(): void
+{
+    (string) env('APP_NAME', 'Laravel');
+}
```

Comment:
There's really no need for this test if the other types test pass because it would already be a string---no need to cast

Reply:
Squashed, dropped that commit and force pushed.